doc/ref/api/manual/sampledata.md

```diff
@@ -0,0 +1,58 @@
+# hvplot.sampledata
+
+The `hvplot.sampledata` module provides convenient access to sample datasets for learning, testing, and prototyping with hvPlot.
+
+## Usage
+
+```python
+import hvplot.pandas  # noqa
+import hvplot.xarray  # noqa
+
+penguins = hvplot.sampledata.penguins("pandas")
+air_temp = hvplot.sampledata.air_temperature("xarray")
+landsat = hvplot.sampledata.landsat_rgb("rioxarray")
+us_states = hvplot.sampledata.us_states("geopandas")
+```
+
+## Installation
+
+The `hvsampledata` package is included as a dependency when installing hvPlot. If you installed hvPlot with minimal dependencies, you may need to install it separately:
+
+::::{tab-set}
+
+:::{tab-item} pip
+
+```bash
+pip install hvsampledata
+```
+
+:::
+
+:::{tab-item} conda
+
+```bash
+conda install conda-forge::hvsampledata
+```
+
+:::
+::::
+
+## Available Datasets
```

Comment:
In the end, I don't find that section very useful and would rather just have a link to hvsampledata's website. For now we could simply link to its repo.